crates/jrsonnet-evaluator/src/integrations/serde.rs

```diff
@@ -46,6 +46,12 @@ impl TryFrom<&Val> for Value {
 	}
 }
 
+impl From<serde_json::Error> for LocError {
+	fn from(e: serde_json::Error) -> Self {
+		LocError::new(RuntimeError(format!("Error parsing JSON {:?}", e).into()))
```

Comment:
Please use `Error` variant for errors (I have some plans for it)

Reply:
Not entirely sure what that meant, but I guess by using `state.evaluate_snippet_raw` that should no longer be a problem